### ISSUE TITLE
JAVA is a Poop 💩

### ISSUE THREAD

**Issue by vkravchik** [extremecodetv/ExtremeCodeOS#219]
Never to give the user possibility to write a code by JAVA

**Comment by BlitDev** [comment]
I agree. I propose to make it possible to write only in muda-muda and ore-ore languages, like C#, F# and Rust. Girly languages like Go and C++ are not worthy of our OS.
